data.js

```diff
@@ -4565,5 +4565,18 @@ module.exports = [
       "Firebase"
     ],
   },
+   {
+    name: "Spacelift",
+    description: "Articles about Terraform, Kubernetes, Ansible, and Docker.",
+    url: "https://spacelift.io/blog",
+    twitter: "@spaceliftio",
+    tags: [
+      "Terraform",
+      "Ansible",
+      "Kubernetes",
+      "Docker",
+      "DevOps",
+      "AWS"
+    ],
```

Comment:
This JSON is broken, there's no closing brace for the object you introduced.

Reply:
Sorry, I must have missed it. All fixed now:)